pages.pt_BR/common/2to3.md

```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversão automática de código Python 2 para Python 3.
+> Mais informação: <https://docs.python.org/3/library/2to3.html>.
+
+- Mostrar as alterações que seriam feitas sem faze-las de fato (simulação):
+
+`2to3 {{diretorio/do/arquivo.py}}`
```

Comment:
```suggestion
`2to3 {{caminho/para/arquivo.py}}`
```
I think `caminho/para/arquivo` is what we use in other pages.

Reply:
Sorry! I'm new a new contributor in general, I've never contributed in other repos 😄  lol

I'll try my best and see other's example to keep the patterns of all translations.

Reply:
don't worry @pedrokpp, we have a few pages with `caminho/do` maybe we should standardize that. Most pages are with `caminho/paga`

Reply:
Either of you know about #5875?